src/app/(views)/(public)/a/[archiveId]/skeleton-loader.tsx

```diff
@@ -0,0 +1,45 @@
+import GridContainer from "~/components/custom/grid-container";
+import { Skeleton } from "~/components/shadcn/skeleton";
+
+export function ArchiveLoadingSkeleton (){
+    return (
+      <div className="space-y-4 pb-6 pt-7">
+        <section className="px-2 md:px-3">
+          <div className="space-y-1">
+            <Skeleton className="h-6 w-64" />
+            <Skeleton className="h-4 sm:w-96 w-64" />
+          </div>
+        </section>
+        <section className="px-0 md:px-3">
+          {Array.from(Array(4).keys()).map((index) => (
+            <GridContainer>
```

Comment:
[Blocking] `index` needs be used as a key, `<GridContainer key={index}>`

Reply:
Thanks for pointing it out. Will fix it.